unix/w0vncserver/portals/RemoteDesktop.cxx

```diff
@@ -51,6 +57,12 @@ core::BoolParameter localCursor("experimentalPortalLocalCursor",
                                 "[EXPERIMENTAL] Render cursor locally",
                                 false);
 
+core::EnumParameter askDisplayChoice("portalAskDisplayChoice",
+                                      "Ask about display choice on connect",
+                                      {"Always", "Once", "Never"}, "Once");
+
+static const char* RESTORE_TOKEN_FILENAME =  "tigervnc.restoretoken";
```

Comment:
We're already in a TigerVNC specific directory, so "restoretoken" should be enough.